spec/models/ability_spec.rb

```diff
@@ -80,6 +80,13 @@
   let(:other_staff_file) { other_staff_scanned_resource.decorate.members.first }
   let(:admin_file) { FactoryBot.build(:file_set, user: admin_user) }
 
+  let(:contributor_ephemera_project) do
+    FactoryBot.create_for_repository(:ephemera_project, member_ids: [contributor_ephemera_folder.id, contributor_ephemera_box.id], contributor_uids: [current_user&.uid])
+  end
+  let(:contributor_ephemera_folder) { FactoryBot.create_for_repository(:ephemera_folder) }
+  let(:contributor_ephemera_box) { FactoryBot.create_for_repository(:ephemera_box, member_ids: contributor_ephemera_folder_in_box.id) }
+  let(:contributor_ephemera_folder_in_box) { FactoryBot.create_for_repository(:ephemera_folder) }
+
```

Comment:
The additions here all make sense.  But I was expecting to see a set of tests that verified that an external contributor could create/update the folders in their project, but not create/update those in a different project, and also that they couldn't see a private non-ephemera item.

Reply:
I can add a set of tests for a different project. There's already tests for non-ephemera items - all the other tests under campus users weren't changed.